native-windows-gui/src/controls/window.rs

```diff
@@ -263,6 +265,24 @@ impl Drop for Window {
     }
 }
 
+#[cfg(feature = "raw-win-handle")]
+unsafe impl HasRawWindowHandle for Window {
+    fn raw_window_handle(&self) -> RawWindowHandle {
+        match self.handle {
+            ControlHandle::Hwnd(hwnd) => {
+                RawWindowHandle::Windows(WindowsHandle {
+                    hwnd: hwnd as *_,
+                    // Get hInstance from hWnd
+                    hinstance: unsafe { GetWindowLongA(hwnd, GWL_HINSTANCE) } as *_
```

Comment:
Tiny fix, use `GetWindowLongW` instead. NWG only use the unicode version of the api call (when it applies).